api-test/integration-tests/src/test/java/com/microsoft/semantickernel/tests/connectors/memory/Hotel.java

```diff
@@ -11,24 +11,37 @@
 public class Hotel {
     @VectorStoreRecordKeyAttribute
     private final String id;
+
     @VectorStoreRecordDataAttribute
     private final String name;
+
     @VectorStoreRecordDataAttribute
     private final int code;
+
     @JsonProperty("summary")
     @VectorStoreRecordDataAttribute()
     private final String description;
-    @JsonProperty("summaryEmbedding")
-    @VectorStoreRecordVectorAttribute(dimensions = 8)
-    private final List<Float> descriptionEmbedding;
+
+    @JsonProperty("summaryEmbedding1")
```

Comment:
are these property names correct?

Reply:
Yes. It's the same vector but with different distance function, they are for testing purposes anyways.